client/src/ratings_reviews/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReviewsList from './ReviewsList.jsx';
+
+class RatingsReviews extends React.Component {
```

Comment:
Please change RatingsReviews to a functional component, the props are the current product id, so there is no need to hold duplicate data in state

Reply:
Change made!